docs/sphinx/user_guide/feature/view.rst

```diff
@@ -85,49 +86,60 @@ access array entries with stride N :subscript:`n` * N :subscript:`(n-1)` * ... *
-arithmetic when their access patterns are the same.
+multiple arrays associated with the MultiView, allowing the arrays to share 
+indexing arithmetic when their access patterns are the same.
 
 The instantiation of a MultiView works exactly like a standard View,
 except that it takes an array-of-pointers. In the following example, a MultiView
-applies a 1-D layout of length 4 to 2 internal arrays in ``myarr``.
+applies a 1-D layout of length 4 to 2 arrays in ``myarr``.
 
 .. literalinclude:: ../../../../examples/multiview.cpp
    :start-after: _multiview_example_1Dinit_start
    :end-before: _multiview_example_1Dinit_end
    :language: C++
 
-The default MultiView accesses internal arrays via the 0th position of the MultiView.
+The default MultiView accesses individual arrays via the 0-th position of the 
+MultiView.
 
 .. literalinclude:: ../../../../examples/multiview.cpp
    :start-after: _multiview_example_1Daccess_start
    :end-before: _multiview_example_1Daccess_end
    :language: C++
 
-The index into the array-of-pointers can be moved to different
-indices of the MultiView ``()`` access operator, rather than the default 0th position. By 
-passing a third template parameter to the MultiView constructor, the internal array index
-and the integer indicating which array to access can be reversed.
+The index into the array-of-pointers can be moved to different argument
+positions of the MultiView ``()`` access operator, rather than the default 
+0-th position. For example, by passing a third template argument to the 
+MultiView constructor in the previous example, the internal array index and 
+the integer indicating which array to access can be reversed.
 
 .. literalinclude:: ../../../../examples/multiview.cpp
    :start-after: _multiview_example_1Daopindex_start
    :end-before: _multiview_example_1Daopindex_end
    :language: C++
 
-As the number of Layout dimensions increases, the index into the array-of-pointers can be
-moved to more distinct locations in the MultiView ``()`` access operator. Here is an example
-which compares the accesses of a 2-D layout on a normal ``RAJA::View`` with a ``RAJA::MultiView``
-with the array-of-pointers index set to the 2nd position.
+With higher dimensional Layouts, the index into the array-of-pointers can be
+moved to other positions in the MultiView ``()`` access operator. Here is an 
+example that compares the accesses of a 2-D layout on a normal ``RAJA::View`` 
+with a ``RAJA::MultiView`` with the array-of-pointers index set to the 2nd 
+position.
  
 .. literalinclude:: ../../../../examples/multiview.cpp
    :start-after: _multiview_example_2Daopindex_start
    :end-before: _multiview_example_2Daopindex_end
    :language: C++
 
-.. note:: MultiView has not yet been tested with atomic accesses. 
+.. note:: ``RAJA::MultiView`` should work with atomic accesses, but we have not yet developed comprehensive tests for this.

```

Comment:
We can remove this if this PR https://github.com/LLNL/RAJA/pull/917 can be squeezed in.

Reply:
Removed.